system/jlib/jtrace.cpp

```diff
@@ -765,14 +766,41 @@ class CTraceManager : implements ITraceManager, public CInterface
                         exporter = opentelemetry::exporter::trace::OStreamSpanExporterFactory::Create();
                         DBGLOG("Tracing to stdout/err...");
                     }
-                    else if (stricmp(exportType.str(), "OTLP")==0)
+                    else if (stricmp(exportType.str(), "OTLP")==0 || stricmp(exportType.str(), "OTLP-HTTP")==0)
+                    {
+                        opentelemetry::exporter::otlp::OtlpHttpExporterOptions trace_opts;
+                        if (exportConfig->hasProp("@url"))
```

Comment:
A more common/simpler way of coding this, avoiding a string copy, would be

```
const char * url = exportConfig->queryProp("@url");
if (url)
   trace_opts.url = url;
```

Same for the endpoint code below.